components/collector/src/source_collectors/sonarqube/security_warnings.py

```diff
@@ -45,8 +45,10 @@ async def _get_source_responses(self, *urls: URL, **kwargs) -> SourceResponses:
                 )
             )
         if "security_hotspot" in security_types:
+            # Note: SonarQube is a bit inconsistent. For issue search, the SonarQube status parameter is called
+            # "statuses", but for hotspots it's called "status".
             api_urls.append(
-                URL(f"{base_url}/api/hotspots/search?projectKey={component}&branch={branch}&statuses=TO_REVIEW&ps=500")
+                URL(f"{base_url}/api/hotspots/search?projectKey={component}&branch={branch}&status=TO_REVIEW&ps=500")
```

Comment:
Overweeg ook status "Acknowledged" mee te nemen

Reply:
That would be a functional change, outside the scope of this bug fix.